common/scripts/mesh-11s.sh

```diff
@@ -107,7 +108,7 @@ network={
     psk="$5"
     key_mgmt=SAE
     ieee80211w=2
-    mesh_fwding=0
+    #mesh_fwding=0
```

Comment:
Do we still have envs where this is not working?

Reply:
As supplicant change is now available in buildroot, we can remove this WAR.
I will raise the change in dev.

Reply:
WAR revert Merged in dev.